tasks/main.yml

```diff
@@ -53,3 +53,14 @@
     - '{{ logrotate_applications }}'
   tags:
     - configuration
+
+- name: Set logrotate hourly state
+  set_fact:
+    hourly_link_state: link
+  when: use_hourly_rotation
+
+- name: Symlink for hourly rotation
```

Comment:
I would do this as follows, so it takes less lines:
```yml
- name: 'symlink for hourly rotation'
  file:
    path: "/etc/cron.hourly/logrotate"
    src: "/etc/cron.daily/logrotate"
    state: "{{ 'link' if logrotate_use_hourly_rotation else 'absent' }}"
```

Reply:
Thanks for review.
Fix in 17cc47173ce9945c74dd3629f6912df504a8eeeb